src/Microsoft.Diagnostics.EventFlow.Inputs.EventFlowTelemetryProcessor/Microsoft.Diagnostics.EventFlow.Inputs.EventFlowTelemetryProcessor.csproj

```diff
@@ -0,0 +1,38 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Yes, and for AI core assembly it would probably work, but it is a very fresh feature. See http://blog.nuget.org/20170316/NuGet-now-fully-integrated-into-MSBuild.html and the comments below.